app/src/main/java/org/fossasia/pslab/fragment/SavedExperiments.java

```diff
@@ -157,6 +158,7 @@ public void onClick(TreeNode node, Object value) {
                     @Override
                     public void onClick(TreeNode node, Object value) {
                         // Open BJT CE Backup experiment
+                        Toast.makeText(getActivity(), "Will be available soon", Toast.LENGTH_LONG).show();
```

Comment:
Make Toast.LENGTH_SHORT in all the Toasts used.

Reply:
Will surely do that @viveksb007